opentelemetry-stdout/src/metrics/exporter.rs

```diff
@@ -266,6 +286,66 @@ fn print_hist_data_points<'a, T: Debug + Copy + 'a>(
     }
 }
 
+fn print_exponential_hist_data_points<'a, T: Debug + Copy + 'a>(
+    data_points: impl Iterator<Item = &'a ExponentialHistogramDataPoint<T>>,
+) {
+    for (i, data_point) in data_points.enumerate() {
+        println!("\t\tDataPoint #{i}");
+        println!("\t\t\tCount          : {}", data_point.count());
+        println!("\t\t\tSum            : {:?}", data_point.sum());
+        if let Some(min) = &data_point.min() {
+            println!("\t\t\tMin            : {min:?}");
+        }
+
+        if let Some(max) = &data_point.max() {
+            println!("\t\t\tMax            : {max:?}");
+        }
+
+        let scale = data_point.scale();
+        let base = 2.0f64.powf(2.0f64.powf(-scale as f64));
+
+        println!("\t\t\tScale          : {}", scale);
+        println!("\t\t\tBase           : {:.3}", base);
+        println!("\t\t\tZeroCount      : {}", data_point.zero_count());
+        println!("\t\t\tZeroThreshold  : {}", data_point.zero_threshold());
+
+        println!("\t\t\tAttributes     :");
+        for kv in data_point.attributes() {
+            println!("\t\t\t\t ->  {}  : {}", kv.key, kv.value.as_str());
+        }
+
+        let negative_bucket = data_point.negative_bucket();
+        let negative_offset = negative_bucket.offset();
+        println!("\t\t\tNegativeOffset : {}", negative_offset);
+        for (i, count) in negative_bucket
+            .counts()
+            .collect::<Vec<_>>()
+            .into_iter()
+            .enumerate()
+            .rev()
+        {
+            let from = -base.powf(i as f64 + negative_offset as f64);
+            let to = -base.powf(i as f64 + negative_offset as f64 + 1.0f64);
+            println!(
+                "\t\t\t\t ->  Bucket {} (<= {:.3} to > {:.3}) : {}",
+                i, from, to, count
```

Comment:
I struggle a bit with the format, but it might just be me:

> Bucket 0 (<= -2.000 to > -4.000) : 5"

Maybe something like is clearer?

> Bucket 0 ( -4.000 < x < -2.00) 

Also i'm not sure of the inclusivity ( x > y <= z ) - when it goes negative - did you have a look at the spec here?